frontend/src/components/dashlist/FlakinessTable.jsx

```diff
@@ -50,10 +49,15 @@ const DashboardList = () => {
     }, [dispatch, seriesId, stabilityChecker, amountOfBuilds, offset]);
 
     return (
-        <TableContainer id="flakiness-table">
-            <StabilityButton value={'unstable'} text="Unstable" />
-            <StabilityButton value={'stable'} text="Stable" />
-            <StyledTable>
+        <TableContainer>
+            <p className="sr-show" role="status" id="flakiness-table-status">
+                {' '}
+            </p>
+            <span role="radiogroup" aria-controls="flakiness-table">
+                <StabilityButton value={'unstable'} text="Unstable" />
+                <StabilityButton value={'stable'} text="Stable" />
+            </span>
+            <StyledTable id="flakiness-table">
```

Comment:
I think the unstable and stable buttons should be somehow outside the table. at least outside the overflow style...
![image](https://user-images.githubusercontent.com/17247485/111778025-beb9aa00-88bc-11eb-9e97-b174c0bc9d09.png)

Reply:
Yes that's true, but I was thinking that this whole table should have its own issue. I don't think it's wise to correct it in this PR because there are so much to do 😬